build/cpp/cmd/CMakeLists.txt

```diff
@@ -50,7 +50,7 @@ set(WRBL_SRCS src/main.cc
 	${CMAKE_BINARY_DIR}/gen/warble.pb.cc)
 
 # Set executable path to bin folder
-set(EXECUTABLE_OUTPUT_PATH bin)
+set(EXECUTABLE_OUTPUT_PATH ${CMAKE_BINARY_DIR}/bin)
```

Comment:
All instances of this are for my convenience. Changed binary folder to main directory.